app/controllers/tours_controller.rb

```diff
@@ -7,9 +7,26 @@ def index
 
   def show; end
 
+  def new
+    @tour = Tour.new
+  end
+
+  def create
+    @tour = Tour.new(tour_params)
```

Comment:
can you add in here
@tour.user = current_user

Reply:
will do!